src/main/java/donnafin/logic/parser/AppendCommandParser.java

```diff
@@ -91,7 +91,12 @@ private AppendCommand parseAsset(ArgumentMultimap argMultimap) throws ParseExcep
             throw new ParseException(Asset.MESSAGE_CONSTRAINTS);
         }
 
-        Asset newAsset = new Asset(name, type, value.toString(), remarks);
+        Asset newAsset = null;
+        try {
+            newAsset = new Asset(name, type, value.toString(), remarks);
+        } catch (IllegalArgumentException e) {
+            throw new ParseException(e.getMessage());
+        }
```

Comment:
Not sure I understand your reasoning for this, would be helpful to add comments here to explain!

Reply:
This was a bug fix for the IllegalArgumentException caused and the output was not shown when a new Asset with no remarks was appended.

This edit ensures that a ParseException is thrown when a remark is not added to the appended asset

Here is the output displayed
![image](https://user-images.githubusercontent.com/77235299/140252989-c66a8a35-7743-4c0a-a1e0-9b79a44580bc.png)

Reply:
Nice, thanks.